monkey/tests/unit_tests/infection_monkey/credential_collectors/test_mimikatz_collector.py

```diff
@@ -110,3 +117,26 @@ def test_pypykatz_result_parsing_no_secrets(monkeypatch):
     collected_credentials = collect_credentials()
     assert len(collected_credentials) == 1
     assert collected_credentials == expected_credentials
+
+
+def test_mimikatz_credentials_stolen_event_published(monkeypatch):
+    mock_event_queue = MagicMock(spec=IEventQueue)
+    mock_subscriber = mock_pypykatz_handler = Mock()
+    monkeypatch.setattr(
+        "infection_monkey.credential_collectors.mimikatz_collector.pypykatz_handler",
+        mock_pypykatz_handler,
+    )
+
+    for event_tag in MIMIKATZ_EVENT_TAGS:
+        mock_event_queue.subscribe_tag(event_tag, mock_subscriber)
+
+    mimikatz_credential_collector = MimikatzCredentialCollector(mock_event_queue)
+    collected_credentials = mimikatz_credential_collector.collect_credentials()
+
+    mock_event_queue.publish.assert_called_once()
+
+    mock_event_queue_call_args = mock_event_queue.publish.call_args[0][0]
+
+    assert isinstance(mock_event_queue_call_args, CredentialsStolenEvent)
+    assert mock_event_queue_call_args.tags == frozenset(MIMIKATZ_EVENT_TAGS)
```

Comment:
Define `MIMIKATZ_EVENT_TAGS` as a `frozenset`.
```suggestion
    assert mock_event_queue_call_args.tags == MIMIKATZ_EVENT_TAGS
```